test/integration/framework/etcd_utils.go

```diff
@@ -41,6 +38,9 @@ func GetEtcdURLFromEnv() string {
 }
 
 func NewEtcdClient() etcd.Client {
+	// Before doing anything else, make sure etcd is available.
+	RequireEtcd()
```

Comment:
yeah, otherwise `init()` breaks the unit tests , even if they run with skipping enabled.